[2024-03-15T01:58:05.747-0400] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: PII_Data_Detection.load_data_from_gcp scheduled__2024-03-15T05:56:00+00:00 [queued]>
[2024-03-15T01:58:05.755-0400] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: PII_Data_Detection.load_data_from_gcp scheduled__2024-03-15T05:56:00+00:00 [queued]>
[2024-03-15T01:58:05.756-0400] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-03-15T01:58:05.817-0400] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): load_data_from_gcp> on 2024-03-15 05:56:00+00:00
[2024-03-15T01:58:05.837-0400] {standard_task_runner.py:60} INFO - Started process 186405 to run task
[2024-03-15T01:58:05.855-0400] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'PII_Data_Detection', 'load_data_from_gcp', 'scheduled__2024-03-15T05:56:00+00:00', '--job-id', '182', '--raw', '--subdir', 'DAGS_FOLDER/src/airflow.py', '--cfg-path', '/tmp/tmpyk_lmdid']
[2024-03-15T01:58:05.877-0400] {standard_task_runner.py:88} INFO - Job 182: Subtask load_data_from_gcp
[2024-03-15T01:58:05.995-0400] {task_command.py:423} INFO - Running <TaskInstance: PII_Data_Detection.load_data_from_gcp scheduled__2024-03-15T05:56:00+00:00 [running]> on host MSI.
[2024-03-15T01:58:06.188-0400] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='PII_Data_Detection' AIRFLOW_CTX_TASK_ID='load_data_from_gcp' AIRFLOW_CTX_EXECUTION_DATE='2024-03-15T05:56:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-15T05:56:00+00:00'
[2024-03-15T01:58:06.193-0400] {logging_mixin.py:188} INFO - environment path is set
[2024-03-15T01:58:06.193-0400] {logging_mixin.py:188} INFO - current project directory /home/vineshgvk/PII-Data
[2024-03-15T01:58:06.194-0400] {logging_mixin.py:188} INFO - destination directory at /home/vineshgvk/PII-Data/dags/processed
[2024-03-15T01:58:06.195-0400] {logging_mixin.py:188} INFO - Trying to create a storage client connection
[2024-03-15T01:58:06.580-0400] {logging_mixin.py:188} INFO - Trying to fetch the specified bucket name
[2024-03-15T01:58:07.345-0400] {logging_mixin.py:188} INFO - Specified bucket is successfully fetched
[2024-03-15T01:58:07.346-0400] {logging_mixin.py:188} INFO - Starting iteration on files from bucket
[2024-03-15T01:58:07.347-0400] {logging_mixin.py:188} INFO - defined the blob to fetch <Blob: pii_data_load, Data/train_com.json, None>
[2024-03-15T01:58:07.348-0400] {logging_mixin.py:188} INFO - local file path is created at /home/vineshgvk/PII-Data/dags/processed/train_com.json
[2024-03-15T01:58:07.349-0400] {logging_mixin.py:188} WARNING - --- Logging error ---
[2024-03-15T01:58:07.352-0400] {logging_mixin.py:188} WARNING - Traceback (most recent call last):
[2024-03-15T01:58:07.354-0400] {logging_mixin.py:188} WARNING -   File "/usr/lib/python3.10/logging/__init__.py", line 1100, in emit
    msg = self.format(record)
[2024-03-15T01:58:07.355-0400] {logging_mixin.py:188} WARNING -   File "/usr/lib/python3.10/logging/__init__.py", line 943, in format
    return fmt.format(record)
[2024-03-15T01:58:07.355-0400] {logging_mixin.py:188} WARNING -   File "/usr/lib/python3.10/logging/__init__.py", line 678, in format
    record.message = record.getMessage()
[2024-03-15T01:58:07.356-0400] {logging_mixin.py:188} WARNING -   File "/usr/lib/python3.10/logging/__init__.py", line 368, in getMessage
    msg = msg % self.args
[2024-03-15T01:58:07.357-0400] {logging_mixin.py:188} WARNING - TypeError: not all arguments converted during string formatting
[2024-03-15T01:58:07.357-0400] {logging_mixin.py:188} WARNING - Call stack:
[2024-03-15T01:58:07.365-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/bin/airflow", line 8, in <module>
    sys.exit(main())
[2024-03-15T01:58:07.366-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/__main__.py", line 57, in main
    args.func(args)
[2024-03-15T01:58:07.367-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
[2024-03-15T01:58:07.367-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/utils/cli.py", line 114, in wrapper
    return f(*args, **kwargs)
[2024-03-15T01:58:07.368-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 438, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
[2024-03-15T01:58:07.369-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 216, in _run_task_by_selected_method
    return _run_task_by_local_task_job(args, ti)
[2024-03-15T01:58:07.369-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 278, in _run_task_by_local_task_job
    ret = run_job(job=job_runner.job, execute_callable=job_runner._execute)
[2024-03-15T01:58:07.370-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/utils/session.py", line 79, in wrapper
    return func(*args, session=session, **kwargs)
[2024-03-15T01:58:07.370-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/jobs/job.py", line 393, in run_job
    return execute_job(job, execute_callable=execute_callable)
[2024-03-15T01:58:07.371-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/jobs/job.py", line 422, in execute_job
    ret = execute_callable()
[2024-03-15T01:58:07.371-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/jobs/local_task_job_runner.py", line 165, in _execute
    self.task_runner.start()
[2024-03-15T01:58:07.372-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/task/task_runner/standard_task_runner.py", line 48, in start
    self.process = self._start_by_fork()
[2024-03-15T01:58:07.373-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/task/task_runner/standard_task_runner.py", line 100, in _start_by_fork
    ret = args.func(args, dag=self.dag)
[2024-03-15T01:58:07.373-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
[2024-03-15T01:58:07.374-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/utils/cli.py", line 114, in wrapper
    return f(*args, **kwargs)
[2024-03-15T01:58:07.374-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 438, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
[2024-03-15T01:58:07.374-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 218, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
[2024-03-15T01:58:07.375-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 297, in _run_raw_task
    return ti._run_raw_task(
[2024-03-15T01:58:07.375-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/utils/session.py", line 79, in wrapper
    return func(*args, session=session, **kwargs)
[2024-03-15T01:58:07.376-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 2360, in _run_raw_task
    self._execute_task_with_callbacks(context, test_mode, session=session)
[2024-03-15T01:58:07.376-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 2532, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
[2024-03-15T01:58:07.377-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 2549, in _execute_task
    return _execute_task(self, context, task_orig)
[2024-03-15T01:58:07.377-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
[2024-03-15T01:58:07.377-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
[2024-03-15T01:58:07.378-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
[2024-03-15T01:58:07.378-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
[2024-03-15T01:58:07.379-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/PII-Data/dags/src/data_download.py", line 63, in load_data_from_gcp_and_save_as_json
    logging.info('works', local_file_path)
[2024-03-15T01:58:07.379-0400] {logging_mixin.py:188} WARNING -   File "/usr/lib/python3.10/logging/__init__.py", line 2138, in info
    root.info(msg, *args, **kwargs)
[2024-03-15T01:58:07.380-0400] {logging_mixin.py:188} WARNING -   File "/usr/lib/python3.10/logging/__init__.py", line 1477, in info
    self._log(INFO, msg, args, **kwargs)
[2024-03-15T01:58:07.380-0400] {logging_mixin.py:188} WARNING -   File "/usr/lib/python3.10/logging/__init__.py", line 1624, in _log
    self.handle(record)
[2024-03-15T01:58:07.381-0400] {logging_mixin.py:188} WARNING -   File "/usr/lib/python3.10/logging/__init__.py", line 1634, in handle
    self.callHandlers(record)
[2024-03-15T01:58:07.381-0400] {logging_mixin.py:188} WARNING -   File "/usr/lib/python3.10/logging/__init__.py", line 1696, in callHandlers
    hdlr.handle(record)
[2024-03-15T01:58:07.381-0400] {logging_mixin.py:188} WARNING -   File "/usr/lib/python3.10/logging/__init__.py", line 968, in handle
    self.emit(record)
[2024-03-15T01:58:07.382-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/utils/log/file_task_handler.py", line 272, in emit
    self.handler.emit(record)
[2024-03-15T01:58:07.382-0400] {logging_mixin.py:188} WARNING - Message: 'works'
Arguments: ('/home/vineshgvk/PII-Data/dags/processed/train_com.json',)
[2024-03-15T01:58:07.586-0400] {logging_mixin.py:188} INFO - File train_com.json downloaded and saved as /home/vineshgvk/PII-Data/dags/processed/train_com.json.
[2024-03-15T01:58:07.586-0400] {python.py:202} INFO - Done. Returned value was: /home/vineshgvk/PII-Data/dags/processed/train_com.json
[2024-03-15T01:58:07.626-0400] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=PII_Data_Detection, task_id=load_data_from_gcp, execution_date=20240315T055600, start_date=20240315T055805, end_date=20240315T055807
[2024-03-15T01:58:07.693-0400] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-15T01:58:07.754-0400] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
